Sources/Clappr/Classes/Base/Core.swift

```diff
@@ -55,18 +56,20 @@ open class Core: UIObject, UIGestureRecognizerDelegate {
     }
 
     @objc open var isFullscreen: Bool = false
-    private var isChromeless: Bool { options.bool(kChromeless) }
+    public var chromelessMode: Bool = false {
+        didSet {
+            updateChromelessModeVisibility()
+            updateGestureRecognizersState()
+        }
+    }
     
     public required init(options: Options = [:], layerComposer: LayerComposer) {
         Logger.logDebug("loading with \(options)", scope: "\(type(of: self))")
 
         self.options = options
         self.layerComposer = layerComposer
         super.init()
-        
-        if !isChromeless {
-            addTapGestures()
-        }
+        addTapGestures()
```

Comment:
It worries me the fact that setting the options (when `kChromeless` is included) affects the way we handle the gestures. As of now, we're updating gestures' state before we actually add them as a result of the `didSet` chain triggered after setting the options. It seems to me that's the reason you're calling `updateChromelessMode` on `render`.

It seems to me that a better option is to actually create the gestures during `render` (which makes sense its purpose is to setup views/view related stuff), and enable/disable the gestures after its creation based on the `chomelessMode` state, which will certainly be set by this point.